### PR TITLE
chore(benchmark): Add best practices for windows systems

### LINKED ISSUE #4207: Benchmarks should also be runnable from Windows systems
**Description**
The benchmarks section describes how to run Zeebe benchmarks. The description is aimed at Linux system. Reproducing these steps on a Windows machine is not a trivial task.

This issue is to document best practices and make changes so that the benchmarks can also be run from a Windows system.

### PR DESCRIPTION
**Pull request by pihme**
## Description

* Adds changes to Dockerfile to set execution permissions when building the Docker image
* Adds best practices to documentation

## Related issues

closes #4207

## Pull Request Checklist

- [ X ] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [ X ] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [ X ] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

File: benchmarks/setup/BestPracticesWindows.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Best Practices Windows
+
+Running the benchmarks is possible with the help of the Windows Subsystem for Linux.
```
Comment (Zelldon) [inline benchmarks/setup/BestPracticesWindows.md:3]: Could we reference here how to setup a subsystem for linux? Any resources do you used?

File: benchmarks/setup/BestPracticesWindows.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+These are the components to install on Windows:
+* Docker
+
+These are the components to install within the WSL:
+* gcloud https://cloud.google.com/sdk/install?hl=de
+* Kubectl https://kubernetes.io/de/docs/tasks/tools/install-kubectl/
+* Helm 3.*  https://helm.sh/docs/intro/install/
+* kubens/kubectx https://github.com/ahmetb/kubectx
+
+When following the instructions, execute all commands that deal with Docker in a Windows shell, and exeucte all other commands in the WSL shell.
```
Comment (Zelldon) [inline benchmarks/setup/BestPracticesWindows.md:16]: :+1:

File: benchmarks/setup/BestPracticesWindows.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Best Practices Windows
```
Comment (Zelldon) [inline benchmarks/setup/BestPracticesWindows.md:1]: There is a doc sub-folder in the benchmark folder, could we move it to that sub-folder OR maybe merge it with the Benchmark readme, where we describe the installation?

File: benchmarks/setup/README.md [hunk 4]
```diff
@@ -2,9 +2,13 @@
...
 Welcome to the setup a benchmark. :wave:
 
+This documnet describes the steps to setup and run a benchmark. It is target at a Linux system.
+
+If you want to run it on Windows, please consider the [Best Practices Windows](BestPracticesWindows.md).
+
 ## Requirements
 
-Make sure you have the following installed: gcloud, kubectl, kubens and helm!
+Make sure you have the following installed: docker, gcloud, kubectl, kubens and helm!
```
Comment (Zelldon) [inline benchmarks/setup/README.md:11]: :+1:

**Review by ChrisKujawa — COMMENTED** [review]
Hey Peter sorry for the late response. Thanks for doing this. Please check my comments below.

File: benchmarks/README.md [hunk 5]
```diff
@@ -48,6 +48,23 @@ curl -LO https://raw.githubusercontent.com/ahmetb/kubectx/master/kubens
 install kubens /usr/local/bin/
 ```
 
+## Best Practices Windows
+
+ unning the benchmarks on Windows is possible with the help of the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```
Comment (Zelldon) [inline benchmarks/README.md:53]:
```suggestion
 Running the benchmarks on Windows is possible with the help of the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```

**Review by ChrisKujawa — APPROVED** [review]
Just a small thing then I think we can merge it

**Pull request merged**